### PR TITLE
🌐 [i18n-KO] Translated encoder-decoder.md to Korean

### PR DESCRIPTION
**Pull request by maximizemaxwell**
# What does this PR do?

Translated the encoder-decoder.md file of the documentation to Korean.
Thank you in advance for your review.

Part of https://github.com/huggingface/transformers/issues/20179
## Before reviewing

  - [x]  Check for missing / redundant translations (번역 누락/중복 검사)
  - [x] Grammar Check (맞춤법 검사)
  - [x] Review or Add new terms to glossary (용어 확인 및 추가)
  - [x]  Check Inline TOC (e.g. [[lowercased-header]])
  - [ ]  Check live-preview for gotchas (live-preview로 정상작동 확인)

## Who can review? (Initial)

@junejae, @Jwaminju, @010kim, @4N3MONE, @boyunJang
## Before submitting

   - [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
   - [x]  Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
    Pull Request section?
   - [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
    to it if that's the case.
   - [x]  Did you make sure to update the documentation with your changes? Here are the
    [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
    [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
   - [ ]  Did you write any new necessary tests?

## Who can review? (Final)

@stevhliu May you please review this PR?

### CODE REVIEW

**Review by stevhliu — APPROVED** [review]
Cool, thanks! We can merge once a PseudoLab team member has reviewed :)

File: docs/source/ko/model_doc/encoder-decoder.md [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+>>> model = TFEncoderDecoderModel.from_encoder_decoder_pretrained(
+...     "./encoder", "./decoder", encoder_from_pt=True, decoder_from_pt=True
+... )
+>>> # This is only for copying some specific attributes of this particular model.
+>>> model.config = _model.config
+```
+
+## 학습[[Training]]
+
+모델이 생성된 후에는 BARY, T5 또는 기타 인코더-디코더 모델과 유사한 방식으로 미세 조정(fine-tuning)할 수 있습니다.
```
Comment (4N3MONE) [inline docs/source/ko/model_doc/encoder-decoder.md:105]:
```suggestion
모델이 생성된 후에는 BART, T5 또는 기타 인코더-디코더 모델과 유사한 방식으로 미세 조정(fine-tuning)할 수 있습니다.
```

File: docs/source/ko/model_doc/encoder-decoder.md [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+...     return_tensors="pt",
+... ).input_ids
+
+>>> labels = tokenizer(
+...     "the eiffel tower surpassed the washington monument to become the tallest structure in the world. it was the first structure to reach a height of 300 metres in paris in 1930. it is now taller than the chrysler building by 5. 2 metres ( 17 ft ) and is the second tallest free - standing structure in paris.",
+...     return_tensors="pt",
+... ).input_ids
+
+>>> # the forward function automatically creates the correct decoder_input_ids
```
Comment (4N3MONE) [inline docs/source/ko/model_doc/encoder-decoder.md:127]:
```suggestion
>>> # forward 함수가 자동으로 적합한 decoder_input_ids를 생성합니다.
```

File: docs/source/ko/model_doc/encoder-decoder.md [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+
+>>> _model = EncoderDecoderModel.from_pretrained("patrickvonplaten/bert2bert-cnn_dailymail-fp16")
+
+>>> _model.encoder.save_pretrained("./encoder")
+>>> _model.decoder.save_pretrained("./decoder")
+
+>>> model = TFEncoderDecoderModel.from_encoder_decoder_pretrained(
+...     "./encoder", "./decoder", encoder_from_pt=True, decoder_from_pt=True
+... )
+>>> # This is only for copying some specific attributes of this particular model.
```
Comment (4N3MONE) [inline docs/source/ko/model_doc/encoder-decoder.md:99]:
```suggestion
>>> # 이 부분은 특정 모델의 구체적인 세부사항을 복사할 때에만 사용합니다.
```

File: docs/source/ko/model_doc/encoder-decoder.md [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+>>> print(generated_text)
+nearly 800 thousand customers were affected by the shutoffs. the aim is to reduce the risk of wildfires. nearly 800, 000 customers were expected to be affected by high winds amid dry conditions. pg & e said it scheduled the blackouts to last through at least midday tomorrow.
+```
+
+## `TFEncoderDecoderModel`에 Pytorch 체크포인트 불러오기[[Loading a PyTorch checkpoint into `TFEncoderDecoderModel`.]]
+
+[`TFEncoderDecoderModel.from_pretrained`] 메서드는 현재 Pytorch 체크포인트를 사용한 모델 초기화를 지원하지 않습니다. 이 메서드에 `from_pt=True`를 전달하면 예외(exception)가 발생합니다. 특정 인코더-디코더 모델에 대한 Pytorch 체크포인트만 존재하는 경우, 다음과 같은 해결 방법을 사용할 수 있습니다:
+
+```python
+>>> # a workaround to load from pytorch checkpoint
```
Comment (4N3MONE) [inline docs/source/ko/model_doc/encoder-decoder.md:88]:
```suggestion
>>> # 파이토치 체크포인트에서 로드하는 해결 방법
```

File: docs/source/ko/model_doc/encoder-decoder.md [hunk 5]
```diff
@@ -0,0 +1,167 @@
...
+
+>>> # let's perform inference on a long piece of text
+>>> ARTICLE_TO_SUMMARIZE = (
+...     "PG&E stated it scheduled the blackouts in response to forecasts for high winds "
+...     "amid dry conditions. The aim is to reduce the risk of wildfires. Nearly 800 thousand customers were "
+...     "scheduled to be affected by the shutoffs which were expected to last through at least midday tomorrow."
+... )
+>>> input_ids = tokenizer(ARTICLE_TO_SUMMARIZE, return_tensors="pt").input_ids
+
+>>> # autoregressively generate summary (uses greedy decoding by default)
```
Comment (4N3MONE) [inline docs/source/ko/model_doc/encoder-decoder.md:76]:
```suggestion
>>> # 자기회귀적으로 요약 생성 (기본적으로 그리디 디코딩 사용)
```

File: docs/source/ko/model_doc/encoder-decoder.md [hunk 6]
```diff
@@ -0,0 +1,167 @@
...
+## 기존 `EncoderDecoderModel` 체크포인트 불러오기 및 추론하기[[Loading an existing `EncoderDecoderModel` checkpoint and perform inference.]]
+
+`EncoderDecoderModel` 클래스의 미세 조정(fine-tuned)된 체크포인트를 불러오려면, Transformers의 다른 모델 아키텍처와 마찬가지로 [`EncoderDecoderModel`]에서 제공하는 `from_pretrained(...)`를 사용할 수 있습니다.
+
+추론을 수행하려면 [`generate`] 메서드를 활용하여 텍스트를 자동 회귀(autoregressive) 방식으로 생성할 수 있습니다. 이 메서드는 탐욕 디코딩(greedy decoding), 빔 서치(beam search), 다항 샘플링(multinomial sampling) 등 다양한 디코딩 방식을 지원합니다.
+
+```python
+>>> from transformers import AutoTokenizer, EncoderDecoderModel
+
+>>> # load a fine-tuned seq2seq model and corresponding tokenizer
```
Comment (4N3MONE) [inline docs/source/ko/model_doc/encoder-decoder.md:64]:
```suggestion
>>> # 미세 조정된 seq2seq 모델과 대응하는 토크나이저 가져오기
```

**Comment by 4N3MONE** [comment]
몇 가지 제안사항 남겨두었습니다. 번역 고생하셨습니다!

**Comment by maximizemaxwell** [comment]
Resolved!🤗
